Dockerfiles/entrypoint_cvp.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# operator-bundle-image validate
+validate_operator_bundle () {
+    echo -e "operatorSDKVersion is $operatorSDKVersion"
+    ansible-playbook -vvvv --connection local -i localhost, operator-test-playbooks/validate-operator-bundle.yml -e"work_dir=/project/output" -e"operator_work_dir=/project/operator-bundle/"
```

Comment:
The empty  annotation was not added for operator bundles.
why do we touch bundle here?

Reply:
I borrowed it from the midstream CI image to include the test. 
I can remove it. It acts as a unit test for ansible-playbooks we have right now. 
I can remove it if you dont want it here.